### DISCUSSION TITLE
How do I tell whether a player is detected by the enemy's radar?

### DISCUSSION THREAD

**Discussion by cyfex** [saul/demofile#240] [Questions] [answered]
Hi saul,

Thank you for your great work. I've been wondering for long about how to tell if a player is detected by the enemy, i. e., is shown in enemy's radar. Is there an event or attribute describing it? It would be helpful for the analysis of player behaviour. Thank you

**Comment by EnriqCG** [comment]
Have you figured it out?

**Comment by cyfex** [comment]
Thank you for your reply. Am I correct by saying: one can do it by iterating through enemy player list and look at the value of isSpottedBy(player) to see if it is spotted by any of them? Is there any other attribute/event to help improve this procedure?

**Comment by saul** [comment, marked as answer]
You can also use `Player#allSpotters` to see which other players have spotted this player.

**Comment by cyfex** [comment]
Get it, thank you.
